libraries/stdlib/samples/test/samples/misc/repeat.kt

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2010-2018 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license
+ * that can be found in the license/LICENSE.txt file.
+ */
+
+package samples.misc
+
+import samples.Sample
+import samples.assertPrints
+
+class Repeat {
+
+    @Sample
+    fun repeatAnyTimes() {
+        var sum10Times = 0
+        repeat(10) {
+            sum10Times += it
+        }
+        assertPrints(sum10Times, "45")
+
+        var sumZeroTimes = 0
+        repeat(0) {
+            sumZeroTimes += it
```

Comment:
You can use some action here that normally fails, to emphasize that the lambda is not executed.